recipe/migrations/jpeg9e.yaml

```diff
@@ -0,0 +1,12 @@
+migrator_ts: 1676157958
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+  # limit the number of prs to do the initial test
+  pr_limit: 1
```

Comment:
Please add a custom PR title here.

Reply:
Via the commit message? or is there an other key.